Lib/test/test_dis.py

```diff
@@ -577,14 +577,10 @@ async def _asyncwith(c):
                 RETURN_CONST             0 (None)
 
 %4d   L12:     CLEANUP_THROW
-
-  --   L13:     JUMP_BACKWARD_NO_INTERRUPT 25 (to L5)
-
-%4d   L14:     CLEANUP_THROW
-
-  --   L15:     JUMP_BACKWARD_NO_INTERRUPT 9 (to L11)
```

Comment:
Previously these jumps had no line number, now they do (and they are the same as the line before and after) so the lineno table is smaller and the dis output is a bit different.